### PR TITLE
Testing knative-function e2e oncluster tests on presumbits

### PR DESCRIPTION
**Pull request by jrangelramos**
Testing knative functions oncluster tests on ci.

/hold

Test only PR

### CODE REVIEW

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

File: ci-operator/config/jrangelramos/func/jrangelramos-func-presumtest.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+tests:
+- as: unit
+  commands: GOFLAGS="" make test
+  container:
+    from: src
+- as: e2e-oncluster-test
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
```
Comment (mgencur) [inline ci-operator/config/jrangelramos/func/jrangelramos-func-presumtest.yaml:35]: Please use `serverless-ci` like we do in https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__415.yaml#L121-L127
We're migrating away from openshift-ci cluster pool.

Also the `releases` section should match version of this cluster pool. Like here: https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__415.yaml#L107-L111

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
https://github.com/openshift/release/pull/50201

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Comment by jrangelramos** [comment]
/pj-rehearse pull-ci-jrangelramos-func-presumtest-e2e-oncluster-test

**Pull request closed without merging**